VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -454,8 +461,10 @@ public void testUnitFragmentCreation() {
     // TODO: Robolectric doesn't have a Gradle-based parameterized test suite, need to implement it.
     @Ignore
     @RunWith(Parameterized.class)
-    public static class ParameterizedFragmentCreation {
-        @Parameterized.Parameters(name= "{index}: testUnitFragmentCreation({0})={1}")
+    public static class ParameterizedFragmentCreation extends UiTest {
+        private static final String DISCUSSIONS_ENABLED = "DISCUSSIONS_ENABLED";
+
+        @Parameterized.Parameters(name = "{index}: testUnitFragmentCreation({0})={1}")
```

Comment:
The first parameter (course unit) doesn't have a good string representation, so you should either add a string that describes it as an additional unused parameter, or just revert to the default naming scheme which includes only the test name and parameterized index.